src/main/resources/openshift/deploy.sh

```diff
@@ -4,6 +4,10 @@ set -e
 
 OCP_PROJECT=rhamt
 
+if [ -z "$RHAMT_VOLUME_CAPACITY" ]; then
+    RHAMT_VOLUME_CAPACITY=10Gi
```

Comment:
Not really... I would think that they should go with some multiple of the input application size (maybe 2.5-3x).